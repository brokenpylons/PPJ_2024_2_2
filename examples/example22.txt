var va = 3;
var vb = 6;
print(va ^ vb);
