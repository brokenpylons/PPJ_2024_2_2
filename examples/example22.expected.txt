define("var") variable("va") assign("=") real("3") term(";") define("var") variable("vb") assign("=") real("6") term(";") print("print") lparen("(") variable("va") pow("^") variable("vb") rparen(")") term(";") 
